rcon/rcon.py

```diff
@@ -467,8 +451,15 @@ def get_detailed_player_info(self, player_name: str) -> GetDetailedPlayer:
         Level: 34
 
         """
+        # Add VIP Status
+        player_data = parse_raw_player_info(raw, player_name)
+        vip_player_ids = set(v[PLAYER_ID] for v in super().get_vip_ids())
+        player_data["is_vip"] = player_data["player_id"] in vip_player_ids
 
-        return parse_raw_player_info(raw, player_name)
+        # Add Profile
+        profile = get_profiles([player_data["player_id"]])
```

Comment:
I think you should use `get_player_profile` which you can call by player ID to avoid the (probably slim) chance of index errors if we get back an empty list from `get_profiles`

I don't have access to your repo so I can't change it 😄